docs/DeveloperGuide.md

```diff
@@ -2,37 +2,207 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+Certain aspects of YMFC's code was inspired from the following member(s) IPs:
+
+[Sanjith](http://github.com/KSanjith/ip)
 
 ## Design & implementation
```

Comment:
The DG might benefit from having a contents page

Reply:
Agreed. Could add a contents section with each of the headings having their own corresponding link